tests/features/ui/ui_admin.feature

```diff
@@ -1,17 +1,17 @@
 @ui
 Feature: Admin UI
-
     Scenario Outline: Admin page view by admin users
         Given user <user> is logged in
         When the user navigates to the admin page
         Then the text Administration is shown
         And Administration option is available in the menu dropdown
         And the banner setup option should be visible
+        And the theme change option should be visible
 
         Examples:
             | user                |
-            | technicaladmin      | # Role: irodsadmin
-            | functionaladminpriv | # Group: priv-admin
+            | technicaladmin      | # admin user as role: irodsadmin
+            | functionaladminpriv | # admin user in group: priv-admin
```

Comment:
Not really needed to fix, but I am wondering if there is a different way to comment this, as the comments are sort of breaking my syntax highlighting of the file.
![image](https://github.com/user-attachments/assets/0c39a3fe-2942-4c6b-a17d-f2397fe31d52)

Reply:
I recommend changing these comments to descriptions. I added an example of how you can do this.

Reply:
Improved the comments, awaiting for the integration test results